generators/app/templates/.eslintrc

```diff
@@ -0,0 +1,9 @@
+---
+parser: babel-eslint
+
+env:
+  es6: true
+  amd: true
+  node: true
+  mocha: true
+  browser: true
```

Comment:
I'll just copy it for now, we can publish it later and maybe even download it on install.